threeML/plugins/FermiGBMTTELike.py

```diff
@@ -18,9 +18,9 @@
 __instrument_name = "Fermi GBM TTE (all detectors)"
 
 
-class FermiGBMLikeTTE(OGIPLike, PluginPrototype):
+class FermiGBMTTELike(OGIPLike, PluginPrototype):
```

Comment:
You don't need double inheritance anymore, since OGIPLike inherits already from PluginPrototype.